packages/theme/nuxt.config.js

```diff
@@ -189,6 +188,7 @@ export default {
       { code: 'USD', label: 'Dollar' },
       { code: 'EUR', label: 'Euro' }
     ],
-    backendUrl: process.env.BACKEND_URL
+    backendUrl: process.env.BACKEND_URL,
+    middlewareUrl: 'http://localhost:3000/api/'
```

Comment:
Where is this used? It looks a bit tricky, we should at least use an env variable here + do some reasonable fallback

Reply:
It's one of the breaking changes - see [here](https://github.com/vuestorefront/vue-storefront/pull/6680). Basically storefront expects this url now in the nuxt config instead of computing it on its own. We already use env variables for the middleware server config in the same file so we can do the same here:
``` js
server: {
  port: process.env.PORT || 3000,
  host: process.env.HOST || '0.0.0.0'
}
```
But first I want to make it work :)